libs/sdk-ui-dashboard/api/sdk-ui-dashboard.api.md

```diff
@@ -5107,6 +5119,31 @@ export type UseDashboardQueryProcessingResult<TQueryCreatorArgs extends any[], T
     run: (...args: TQueryCreatorArgs) => void;
 };
 
+// @alpha
+export const useDashboardScheduledEmails: () => {
+    isScheduledEmailingVisible: boolean;
+    enableInsightExportScheduling: boolean;
+    defaultOnScheduleEmailing: () => void;
+    openScheduleEmailingDialog: () => {
```

Comment:
This should likely return void? Also what about hiding `setScheduledEmailDefaultAttachment` under `openScheduleEmailingDialog` param? Is there some use case to change the default attachment, other than before opening/when closing it?

Reply:
Good point. There is no other use case yet, so we can hide it 👍